### PR TITLE
Parse `TEST_INSTANCE_PARAMS`

### PR DESCRIPTION
**Pull request by ackintosh**
Resolves https://github.com/testground/sdk-rust/issues/18

### CODE REVIEW

**Review by mxinden — COMMENTED** [review]
Cool. This looks good to me. Thanks. Just needs a changelog entry and a patch version bump.

**Comment by ackintosh** [comment]
Added a changelog entry. Please point out if something is wrong since my English isn't very good. 😅

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Change
+- Change `RunParameters::test_instance_params` from String to HashMap which contains key-value pairs from parsing the parameter string. See [PR 19].
```
Comment (mxinden) [inline CHANGELOG.md:9]:
```suggestion
- Change `RunParameters::test_instance_params` from `String` to `HashMap` which contains key-value pairs from parsing the parameter string. See [PR 19].
```

**Review by mxinden — COMMENTED** [review]
:pray: 

I missed that this is a breaking change. Would you mind bumping the `Cargo.toml` version to `v0.3.0`?

**Comment by ackintosh** [comment]
Thanks for the review! I bumped up the version to `v0.3.0`.

**Comment by mxinden** [comment]
@ackintosh let me know in case you want me to cut `v0.3.0` right away.

**Comment by ackintosh** [comment]
@mxinden Could you cut a new release when you have time? I would use the latest version in https://github.com/sigp/gossipsub-testground. 😃

**Comment by mxinden** [comment]
@ackintosh for sure. Though should we get https://github.com/testground/sdk-rust/pull/22 in first?

**Comment by ackintosh** [comment]
Yes, please, after #22 has been merged. 🙏

**Pull request merged**